tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,11 @@
+Exit Code: 1
+Standard output:
+lib/ExternalModule.js(305,28): error TS2554: Expected 1 arguments, but got 0.
+lib/Stats.js(68,31): error TS2554: Expected 3 arguments, but got 2.
```

Comment:
context object is technically optional although all the other calls provide a starting context